conf/igenomes.config

```diff
@@ -9,184 +9,413 @@
 
 params {
   // illumina iGenomes reference file paths
-  // TODO nf-core: Add new reference types and strip out those that are not needed
   genomes {
     'GRCh37' {
```

Comment:
you could remove this config if you are not using it

Reply:
done